src/main/java/io/strimzi/test/container/StrimziKafkaContainer.java

```diff
@@ -280,6 +282,21 @@ private String writeOverrideString(Map<String, String> kafkaConfigurationMap) {
         return kafkaConfiguration.toString();
     }
 
+    private String randomUuid() {
```

Comment:
Please could you add a comment explaining that we're rolling our own uuid generation here, and replicating Kafka's logic to avoid the dependency.

Reply:
Thanks for the approval and feedback. I added a comment. Let me know if this is what you were expecting (English is not my native tong, I hope this is understandable).